docker-compose.yml

```diff
@@ -31,13 +52,20 @@ services:
       - es_data:/usr/share/elasticsearch/data
     ports:
       - "9200:9200"
+    healthcheck:
+      test: curl -s http://localhost:9200 >/dev/null || exit 1
```

Comment:
maybe nitpicky, but is there a reason to use curl here and wget elsewhere?   and that this one has a /dev/null?

Reply:
Excellent point, I don't see any reason not to use the same format as for the other healthcases to have that more consistent. Changing that in the next commit.